atlas/socket/execute_test.go

```
@@ -0,0 +1,128 @@
+	}
+}
+
+func TestExecute_Handle(t *testing.T) {
+	conn, err := sqlite.OpenConn(":memory:", 0)
+	if err != nil {
+		t.Fatalf("Failed to open SQLite connection: %v", err)
+	}
+	defer conn.Close()
+
+	existing := conn.Prep("SELECT 1")
+	defer existing.Finalize()
```

golangci-lint (1.62.2)

Error return value of `existing.Finalize` is not checked

(errcheck)